observability/otel/otel-collector/otel-collector-processors.adoc

```diff
@@ -430,3 +430,26 @@ config: |
 <2> Defines a value provided in the `metrics` field as a `strict` exact match or `regexp` regular expression.
 <3> Lists the metric names, which are exact matches or matches for regular expressions, of the metrics to be converted to delta metrics. If a metric matches both the `include` and `exclude` filters, the `exclude` filter takes precedence.
 <4> Optional: Configures which metrics to exclude. When omitted, no metrics are excluded from conversion to delta metrics.
+
+[id="groupbyattrsprocessor-processor_{context}"]
+== Group-by-Attributes Processor
+
+This processor groups all spans, log records, and metric datapoints that share the same attributes by reassigning them to a Resource that matches those attributes.
+
+:FeatureName: The Group-by-Attributes Processor
+include::snippets/technology-preview.adoc[]
+
+Configuring this processor involves at minimum specifying an array of attribute keys to be used to group spans, log records, or metric datapoints together, as in the following example:
+
+[source,yaml]
+----
+# ...
+processors:
+  groupbyattrs:
+    keys: # <1>
+      - <key1> # <2>
+      - <key2>
+# ...
+----
+<1> Specifies attribute keys to group by.
+<2> If a processed span, log record, or metric datapoint contains at least one of the specified attribute keys, it is reassigned to a Resource that shares the same attribute values; and if no such Resource exists, a new one is created. Also, if none of the specified attribute keys is present in the processed span, log record, or metric datapoint, then it remains associated with its current Resource. Multiple instances of the same Resource are consolidated.
```

Comment:
```suggestion
<2> If a processed span, log record, or metric datapoint contains at least one of the specified attribute keys, it is reassigned to a Resource that shares the same attribute values. If no such Resource exists, a new one is created. If none of the specified attribute keys is present in the processed span, log record, or metric datapoint, then it remains associated with its current Resource. Multiple instances of the same Resource are consolidated.
```

Reply:
Thank you. I kept the first sentence as is (for better readability despite extra length) and updated the second sentence as you suggested.